### ISSUE TITLE
[Bug]: example app fails

### ISSUE THREAD

**Issue by chlebowa** [insightsengineering/teal.modules.general#505]
### What happened?

There are two example apps in `tm_g_distribution` and the first one ("non-clinical data) runs like this:

![image](https://user-images.githubusercontent.com/114988527/210214812-d618fe21-27ae-4ca0-ba61-e223e6cbf3ed.png)

```
Warning: Error in : no applicable method for 'depth' applied to an object of class "NULL" 
 when evaluating qenv code:
g <- ggplot(ANL, aes(Petal.Length)) + geom_histogram(position = "identity", aes_string(y = "..density.."), bins = 30, alpha = 0.3) + stat_density(aes_string(y = "..density.."), geom = "line", position = "identity", alpha = 0.5, size = 2, n = 512) + ggplot2::labs(subtitle = "Plot generated by Distribution Module") + ggplot2::theme_gray()
print(g)
  226: <Anonymous>
  225: stop
  224: [[.qenv.error
  222: <reactive>
  220: .func
  217: contextFunc
  216: env$runWith
  209: ctx$run
  208: self$.updateValue
  206: plot_r
  204: <reactive>
  202: .func
  199: contextFunc
  198: env$runWith
  191: ctx$run
  190: self$.updateValue
  188: plot_type
  187: <reactive>
  185: .func
  182: contextFunc
  181: env$runWith
  174: ctx$run
  173: self$.updateValue
  171: plot_reactive
  170: renderPlot
  168: func
  128: drawPlot
  114: <reactive:plotObj>
   98: drawReactive
   85: renderFunc
   84: output$teal-main_ui-root-distribution_module-hist_plot-plot_main
    3: runApp
    2: print.shiny.appobj
    1: <Anonymous>
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

**Comment by gogonzo** [comment]
From @nikolas-burkoff [comment](https://github.com/insightsengineering/teal.modules.general/pull/498#discussion_r1060601662)
![image](https://user-images.githubusercontent.com/6959016/210727405-2a68e2b9-8796-407a-b84f-63513b67d5f5.png)

Reason this happens is that some tests require strata which is optional. PR incoming...

**Comment by gogonzo** [comment]
Initial error not reproducible anymore. Second issue mentioned by @nikolas-burkoff closed in #506
